go/teams/service_helper.go

```diff
@@ -835,14 +835,25 @@ func RequestAccess(ctx context.Context, g *libkb.GlobalContext, teamname string)
 	return ret, err
 }
 
-func TeamAcceptInviteOrRequestAccess(ctx context.Context, g *libkb.GlobalContext, tokenOrName string) error {
+func TeamAcceptInviteOrRequestAccess(ctx context.Context, g *libkb.GlobalContext, tokenOrName string) (keybase1.TeamAcceptOrRequestResult, error) {
 	// First try to accept as an invite
 	err := AcceptInvite(ctx, g, tokenOrName)
-	if err != nil {
-		// Failing that, request access as a team name
-		_, err = RequestAccess(ctx, g, tokenOrName)
+	if err == nil {
+		return keybase1.TeamAcceptOrRequestResult{
+			WasToken: true,
```

Comment:
What does `Open` mean? If it's whether the team is open, then it's missing here. If it's whether the request that just sent was an open team request then ok but it deserves a comment or name change

Reply:
It's used to indicate whether RequestAccess(teamname) had to send request to team admins that they have to approve manually; or if it's an open team and user will be added automatically. What would be a better name for that?

Reply:
Could be `WasOpenRequest`. But you could just throw that comment in avdl
``
// Whether a request was sent to team admins that they will automatically approve.
```